core/pta/system.c

```diff
@@ -311,11 +311,11 @@ static void close_session(void *sess_ctx)
 {
 	struct system_ctx *ctx = sess_ctx;
 
-	handle_db_destroy(&ctx->db, ta_bin_close);
+	handle_db_destroy(&ctx->db, bin_close);
```

Comment:
All the cleanup in this function should probably be removed, together with the `*sess_ctx` initialization in `open_session()` above.

Reply:
These are removed in the next commit, `core: remove sess_ctx handling from system PTA`.
Do you suggest to squash that into this?

Reply:
Yes please.

Reply:
Ack

Reply:
Done